tests/connect/test_analyze_plan.py

```diff
@@ -0,0 +1,7 @@
+from __future__ import annotations
+
+
+def test_analyze_plan(spark_session):
+    data = [[1000, 99]]
+    df1 = spark_session.createDataFrame(data, schema="Value int, Total int")
+    df1.collect()
```

Comment:
do we have `df.schema()` implemented yet? if so, we could assert that the schema matches an expected schema. 

I'd also try with something that wouldn't implicitly get coerced to that type. like try using `long` instead of `int` and make sure that the supplied schema is used instead of the implied schema.